tenable/ot/graphql/schema/plugins.py

```diff
@@ -0,0 +1,69 @@
+from marshmallow import fields
+
+from tenable.ot.graphql.definitions import NodesSchema, SchemaBase
+from tenable.ot.schema.plugins import PluginDetails, Plugin, Plugins, PluginRef
+
+
+class PluginRefSchema(SchemaBase):
+    dataclass = PluginRef
+
+    name = fields.String(required=True)
+    value = fields.String(required=True)
+    url = fields.String(required=True)
+
+
+class PluginDetailsSchema(SchemaBase):
+    dataclass = PluginDetails
+
+    id = fields.Integer(required=True)
+    name = fields.String(required=True)
+    source = fields.String(required=True)
+    family = fields.String(required=True)
+    description = fields.String(required=True)
+    solution = fields.String(required=True)
+    see_also = fields.List(
+        fields.String(required=True), required=True, data_key="seeAlso"
+    )
+    plugin_type = fields.String(allow_none=True, data_key="pluginType")
+    plugin_pub_date = fields.String(allow_none=True, data_key="pluginPubDate")
+    plugin_mod_date = fields.String(allow_none=True, data_key="pluginModDate")
+    vuln_pub_date = fields.String(allow_none=True, data_key="vulnPubDate")
+    vuln_mod_date = fields.String(allow_none=True, data_key="vulnModDate")
```

Comment:
Shouldn't these be returned as datetime objects?

Reply:
fixed